packages/desktop/views/update-stronghold/views/UpdateStrongholdView.svelte

```diff
@@ -33,8 +35,17 @@
                 await migrateStrongholdFromOnboardingProfile(password)
                 emitStrongholdMigrationEvent({ success: true, onboardingType })
             } else {
-                await migrateStrongholdFromActiveProfile(password)
-                emitStrongholdMigrationEvent({ success: true })
+                try {
+                    await migrateStrongholdFromActiveProfile(password)
+                    emitStrongholdMigrationEvent({ success: true })
+                } catch (err) {
+                    const message = err?.message ?? ''
+                    if (message.includes('input snapshot')) {
```

Comment:
these 2 strings look a bit generic, is this the best match we can make?

Reply:
No, we will improve this